### PR TITLE
Fixes alien runtime spam

### PR DESCRIPTION
**Pull request by AnturK**

### CODE REVIEW

**Comment by JJRcop** [comment]
who-thought-this-was-a-good-idea
Someone who didn't understand short circuiting, or the fact dna may be null.

File: code/modules/mob/living/carbon/life.dm [hunk 1]
```diff
@@ -412,8 +412,8 @@
 			liver_failure()
 		else
 			liver.failing = FALSE
-
-	if(((!(NOLIVER in dna.species.species_traits)) && (!liver)))
+
```
Comment (praisenarsie) [inline code/modules/mob/living/carbon/life.dm]: disgusting whitespace

Comment (AnturK) [reply code/modules/mob/living/carbon/life.dm]: Oh whoops, will remove in a sec

**Comment by XDTM** [comment]
I also fixed this bug in #30572, if someone feels like merging it first

**Pull request closed without merging**